packages/compass-import-export/src/import/import-json.ts

```diff
@@ -75,8 +75,9 @@ export async function importJSON({
         const doc = EJSON.deserialize(chunk.value);
         if (process.env.COMPASS_E2E_TEST_IMPORT_ABORT_TIMEOUT === 'true') {
           setTimeout(() => {
+            // Give the test more than enough time to click the abort before we continue.
             callback(null, doc);
-          }, 2000);
+          }, 10000);
```

Comment:
I think the flakyness was just caused by 2 seconds not being enough time for the test to click the abort button.